api-v2/beacon/blocks/block.yaml

```diff
@@ -0,0 +1,48 @@
+get:
+  operationId: getBlock
+  summary: Get block
+  description: Retrieves block details for given block id.
+  tags:
+    - Beacon
+  parameters:
+    - name: block_id
+      in: path
+      $ref: '../../../beacon-node-oapi.yaml#/components/parameters/BlockId'
+  responses:
+    "200":
+      description: Success
+      content:
+        application/json:
+          schema:
+            title: GetBlockResponse
+            type: object
+            properties:
+              version:
+                type: string
+                enum: [phase0, altona]
```

Comment:
```suggestion
                enum: [phase0, altair]
```

Reply:
thanks, i had it in my mind it was altona, and changed most but missed this one, pushed up the change.